### PR TITLE
Adding quote in white spaced args

### LINKED ISSUE #7: Quoting for filenames with spaces
Neat idea!

On my Ubuntu installation, something like `ls | ipt |cat` has problems on file names containing spaces.  Is there an easy way to add an option to ipt to cause it to quote the output, e.g. as bash ``printf '%q' ` would do?

### PR DESCRIPTION
**Pull request by andrefarzat**
closes #7 

I faced this issue today and saw that the fix would be easy 😸

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -37,18 +38,28 @@ module.exports = function ipt(p, ttys, log, options, input, error) {
 		p.exit(0);
 	}
 
+	function formatResult(str) {
+		if (options.unquoted) {
+			return str;
+		}
+
+		return str.indexOf(' ') >= 0 ? `"${str}"` : str;
```
Comment (ruyadorno) [inline src/index.js]: oops, the codebase is es5 only 😊

File: src/index.js [hunk 2]
```diff
@@ -37,18 +38,28 @@ module.exports = function ipt(p, ttys, log, options, input, error) {
...
+	}
+
 	function onPrompt(answer) {
+		var result = typeof answer.stdin === 'string' ? formatResult(answer.stdin) : answer.stdin.map(formatResult);
```
Comment (ruyadorno) [inline src/index.js]: very minor, care to break into multiple lines? like:

```js
var result = test ?
  thing :
  otherthing;
```

Comment (andrefarzat) [inline src/index.js]: sure like that ? why not like:
```js
var result = test 
  ? thing
  : otherthing;
```

Even thou, I pushed in the way you asked 😸

**Comment by ruyadorno** [comment]
awesome 🎉 

👍 many thanks!

**Pull request merged**